pages/tipsy/game.jsx

```diff
@@ -129,15 +135,29 @@ const Game = ({ currentGame }) => {
                     onPress={() => tilt("west", game.currentPlayer)}
                 >
                     <View style={styles.leftArrow}>
-                        <Text>◄</Text>
+                        <noscript>
```

Comment:
Nice! It might be a good idea to extract the `TouchableOpacity` component and its content to a reusable component though.

Reply:
I do agree